packages/@vue/cli-plugin-e2e-nightwatch/package.json

```diff
@@ -29,10 +29,12 @@
     "nightwatch": "^1.2.2"
   },
   "devDependencies": {
-    "selenium-server": "^3.141.59",
     "chromedriver": "^76.0.1",
     "geckodriver": "^1.16.2"
   },
+  "optionalDependencies": {
```

Comment:
Optional peer dependencies are listed in "peerDependencies" (so that they won't be installed when running `npm install`) along with a "peerDependenciesMeta" field (to suppress the "missing peer dependencies" warning).

https://github.com/yarnpkg/rfcs/blob/master/accepted/0000-optional-peer-dependencies.md

```
{
  "peerDependencies": {
    "selenium-server": "^3.141.59"
  },
  "peerDependenciesMeta": {
    "selenium-server": {
      "optional": true
    }
  }
}
```

"optionalDependencies" will still be installed unless users explicitly specify the `--no-optional` flag.

Reply:
Cool, thanks for the details.